### PR TITLE
Removed unused variables (#6704)

### LINKED ISSUE #6704: [CAL-923] chore: remove defined but never used variables
CleanShot 2023-01-25 at 18 56 46@2x

<sub>[CAL-923](https://linear.app/calcom/issue/CAL-923/chore-remove-defined-but-never-used-variables)</sub>

### PR DESCRIPTION
**Pull request by blam404**
## What does this PR do?

Updated and removed unused variables. There were some variables that looked like it could be used for future things so I either commented them out or left them as is. Still new to this project so I'm not 100% sure about what some things do and didn't want to risk removing something that might be necessary in the future.

Fixes #6704 

**Environment**: Staging(main branch) / Production

## Type of change

- [ ] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- [ ] No testing required. Removed unused variables should not have an effect on anything.

### CODE REVIEW

File: apps/web/components/getting-started/components/ConnectedCalendarItem.tsx [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import { useLocale } from "@calcom/lib/hooks/useLocale";
-
+//import { useLocale } from "@calcom/lib/hooks/useLocale";
```
Comment (zomars) [inline apps/web/components/getting-started/components/ConnectedCalendarItem.tsx:1]: We can safely remove these.

File: apps/web/components/getting-started/components/ConnectedCalendarItem.tsx [hunk 2]
```diff
@@ -21,7 +20,7 @@ interface IConnectedCalendarItem {
 
 const ConnectedCalendarItem = (prop: IConnectedCalendarItem) => {
   const { name, logo, externalId, calendars, integrationType } = prop;
-  const { t } = useLocale();
+  //const { t } = useLocale();
```
Comment (zomars) [inline apps/web/components/getting-started/components/ConnectedCalendarItem.tsx:23]: And this

File: apps/web/playwright/auth/auth-index.e2e.ts [hunk 3]
```diff
@@ -1,7 +1,5 @@
-import { expect } from "@playwright/test";
-
-import { WEBAPP_URL } from "@calcom/lib/constants";
-
+// import { expect } from "@playwright/test";
+// import { WEBAPP_URL } from "@calcom/lib/constants";
```
Comment (zomars) [inline apps/web/playwright/auth/auth-index.e2e.ts:2]: Let's remove instead of commenting

File: packages/ui/form/Select.tsx [hunk 4]
```diff
@@ -66,7 +66,7 @@ export function UnstyledSelect<
   Option,
   IsMulti extends boolean = false,
   Group extends GroupBase<Option> = GroupBase<Option>
->({ className, ...props }: SelectProps<Option, IsMulti, Group>) {
+>({ ...props }: SelectProps<Option, IsMulti, Group>) {
```
Comment (zomars) [inline packages/ui/form/Select.tsx:69]: Let's revert this. I think we don't want to pass `className` with the rest of `...props`. In these cases removing `className` will affect `props` structure.

**Review by zomars — DISMISSED** [review]
Thank you for your contribution 🙏 

Left some comments, also can you please skip `yarn.lock` for this PR?

**Comment by blam404** [comment]
@zomars I updated the changes based on your comments.

**Review by emrysal — APPROVED** [review]
Files inspected, check-types handles the rest, thanks for your contribution!

**Pull request merged**